prepare/cards/scigen.py

```diff
@@ -0,0 +1,59 @@
+from unitxt.blocks import (
+    ConstructTableFromRowsCols,
+    LoadHF,
+    RenameFields,
+    SerializeTableAsIndexedRowMajor,
+    Set,
+    TaskCard,
+)
+from unitxt.catalog import add_to_catalog
+from unitxt.operators import FilterByCondition
+from unitxt.task import Task
+from unitxt.templates import InputOutputTemplate, TemplatesList
+from unitxt.test_utils.card import test_card
+
+card = TaskCard(
+    loader=LoadHF(path="kasnerz/scigen", data_classification_policy=["public"]),
+    preprocess_steps=[
+        FilterByCondition(values={"table_content_values": "[]"}, condition="ne"),
+        ConstructTableFromRowsCols(
+            fields=["table_column_names", "table_content_values"],
+            to_field="table",
+        ),
+        SerializeTableAsIndexedRowMajor(field_to_field=[["table", "input"]]),
+        Set({"type_of_input": "table"}),
+        Set({"type_of_output": "text"}),
+        RenameFields(field_to_field={"text": "output", "table_caption": "caption"}),
+    ],
+    task=Task(
```

Comment:
@csrajmohan can you change the task here to this one:
https://github.com/IBM/unitxt/blob/dcbdb31348066ffccd2de4482902c105744ea72e/prepare/cards/numeric_nlg.py#L15-L28

This will allow users to explore in the interactive UI (works only with catalog tasks) and also I think a better way to define things